### PR TITLE
Add deprecation info to README

### PR DESCRIPTION
**Pull request by julienmartinlevrai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
+* [Forum post for the change](https://forum.makerdao.com/t/chainlog-improving-how-we-keep-track-of-our-deployed-contracts/11834)
+* [Chainlog contract’s source code](https://github.com/makerdao/dss-chain-log)
+* [Chainlog contract’s mainnet deployment](https://etherscan.io/address/0xda0ab1e0017debcd72be8599041a2aa3ba7e740f)
+* [UI and API’s source code](https://github.com/makerdao/chainlog-ui)
+* [UI](https://chainlog.makerdao.com)
+* [API](https://chainlog.makerdao.com/api.html)
+
+# Old README
+
+## Maker Foundation Multi-Collateral DAI Public Releases Page
```
Comment (naszam) [inline README.md:14]: I'd suggest something like the following, if the repo won't be archived.
```suggestion
## Maker Foundation Multi-Collateral DAI Public Releases Page [Deprecated]
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,21 @@
-# Maker Foundation Multi-Collateral DAI Public Releases Page
+# Deprecation Notice
+
+MakerDAO’s Changelog was deprecated in favor of its on-chain counterpart, the Chainlog. Please refer to the following resources:
+
+* [Forum post for the change](https://forum.makerdao.com/t/chainlog-improving-how-we-keep-track-of-our-deployed-contracts/11834)
+* [Chainlog contract’s source code](https://github.com/makerdao/dss-chain-log)
+* [Chainlog contract’s mainnet deployment](https://etherscan.io/address/0xda0ab1e0017debcd72be8599041a2aa3ba7e740f)
+* [UI and API’s source code](https://github.com/makerdao/chainlog-ui)
```
Comment (naszam) [inline README.md:8]:
```suggestion
* [Chainlog UI and API’s source code](https://github.com/makerdao/chainlog-ui)
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+* [UI](https://chainlog.makerdao.com)
+* [API](https://chainlog.makerdao.com/api.html)
```
Comment (naszam) [inline README.md:10]:
```suggestion
* [Chainlog historical API](https://chainlog.makerdao.com/api.html)
```

**Comment by julienmartinlevrai** [comment]
@naszam thank you for your review, I applied your suggestions

**Review by naszam — APPROVED** [review]
LGTM

**Review by zdumitru — APPROVED** [review]

**Pull request merged**